numpy/ma/core.py

```diff
@@ -28,12 +28,14 @@
 
 import numpy as np
 import numpy.core.umath as umath
+from numpy.core import multiarray as mu
```

Comment:
At some point I'd really like to get rid of most of these imports, plain old `np.*` seems to work in most cases.